app/authentication/authenticator.py

```diff
@@ -21,18 +22,18 @@ def authenticate(headers):
 
     else:
         res = Response(response="Authorization header required.", status=400, mimetype="text/html")
-        logger.debug("Authorization header not supplied.")
+        logger.debug('Authorization header not supplied.')
         return res
 
 
 def check_jwt(token):
     JWT_ENCRYPT = settings.SM_JWT_ENCRYPT
-    logger.debug('JWT Encryption (0=disabled, 1=enabled) : {}'.format(JWT_ENCRYPT))
+    logger.debug('JWT Encryption (0=disabled, 1=enabled)', JWT_ENCRYPT=JWT_ENCRYPT)
     try:
         if JWT_ENCRYPT == '1':
             decrypter = Decrypter()
             decrypted_jwt_token = decrypter.decrypt_token(token)
-            logger.debug("Decrypted JWT.")
+            logger.debug('Decrypted JWT.')
```

Comment:
Doesn't give any context. Also we don't normally finished messages with a period '.'